cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/executor/ExecutionTaskPlanner.java

```diff
@@ -418,8 +417,8 @@ private void maybeAddLeaderChangeTasks(Collection<ExecutionProposal> proposals,
    * Clear all the states.
    */
   public void clear() {
-    _intraPartMoveTaskByBrokerId.clear();
-    _interPartMoveTaskByBrokerId.clear();
+    _intraPartMoveTasksByBrokerId.clear();
+    _interPartMoveTasksByBrokerId.clear();
```

Comment:
Should we also clear `_interPartMoveBrokerIds`?

Reply:
@zornhsu Would it be safer to call `.clear()` on `_interPartMoveBrokerIds` rather than setting the to `null`?
With the former option, a call to the public function `getInterBrokerReplicaMovementTasks(...)` would be safe, but the latter option can cause a NPE.

Reply:
Yeah, I was debating how we should do this. The difference was that `_interPartMoveBrokerIds` is initiated each time when we pass in a Strategy. 
Directly calling `.clear()` would fail `taskManager.clear()` in `ExecutionTaskManagerTest#testStateChangeSequences`.
In addition, A Strategy S2 which is passed in currently might not be the same as the previous Strategy S1. This was the main reason I ended up using null to force user to create a new `_interPartMoveBrokerIds ` with their new Strategy S2.